internal/services/containers/container_group_resource_test.go

```diff
@@ -603,7 +622,7 @@ func TestAccContainerGroup_emptyDirVolumeShared(t *testing.T) {
 				check.That(data.ResourceName).ExistsInAzure(r),
 			),
 		},
-		data.ImportStep("ip_address_type"),
+		data.ImportStep("ip_address_type", "dns_name_label_reuse_policy"),
```

Comment:
Do we have a reason to ignore the `dns_name_label_reuse_policy` here?

Reply:
@magodo I'm blocked on this one.
The problem is that if we set `Default:  string(containerinstance.DnsNameLabelReusePolicyUnsecure)` for `dns_name_label_reuse_policy`, then tests fail, because we expect to get `dns_name_label_reuse_policy` from created resources, while it is missing when `ip_address_type == None`. 
If I remove `Default` for `dns_name_label_reuse_policy`, then it complains the other side. That template was not having  an explicit definition, but created resource returns the value.
I'm trying to find a way to make `Default` act differently, depending on `ip_address_type` value.

Reply:
Oh, I think  have managed to resolve it. Combination of Set/Get in *Read/*Create methods...